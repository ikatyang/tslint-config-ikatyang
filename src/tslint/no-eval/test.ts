declare const something: any;

// good

const good1 = something;

// bad

const bad1 = eval("something");
