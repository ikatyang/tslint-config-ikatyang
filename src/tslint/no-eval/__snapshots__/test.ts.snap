// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
declare const something: any;

// good

const good1 = something;

// bad

const bad1 = eval("something");
             ~~~~               [forbidden eval]

`;
